jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/registry/applications/AliasEndpoint.java

```diff
@@ -117,20 +123,43 @@ public Response crossOriginForInstallations(@Context HttpHeaders headers, @PathP
 	 *             required values).
 	 * @statuscode 401 The request requires authentication.
 	 *
-	 *             TODO - Rename to registered
 	 */
 	@GET
 	@Path("/exists/{alias}")
 	@Produces(MediaType.APPLICATION_JSON)
 	@ReturnType("java.lang.Boolean")
-	public Response registered(@PathParam("alias") String alias, @Context HttpServletRequest request) {
-		if (aliasService.registered(alias))
-			return appendAllowOriginHeader(Response.ok().entity(Boolean.TRUE), request);
+	@Deprecated
```

Comment:
Can you please add comment explaining why is this API deprecated and which API should the client use instead?

Reply:
DONE